xdsl/dialects/func.py

```diff
@@ -67,10 +67,11 @@ class Call(Operation):
     # TODO how do we verify that the types are correct?
 
     @staticmethod
-    def get(callee: Union[str, FlatSymbolRefAttr],
-            operands: List[Union[SSAValue, Operation]],
+    def get(callee: Union[str, FlatSymbolRefAttr], ops: List[Union[SSAValue,
+                                                                   Operation]],
             return_types: List[Attribute]) -> Call:
-        return Call.build(operands=[operands],
+        ops = list(ops)
```

Comment:
do you need this? Is it just to copy the list?

Reply:
right, thanks!